plugins/index.js

```diff
@@ -134,7 +133,7 @@ export function removePlugin(manifest) {
     }
     unregisterAllPluginWebSocketEvents(manifest.id);
     unregisterPluginReconnectHandler(manifest.id);
-    unregisterAdminConsolePlugin(manifest.id);
+    PluginRegistry.unregisterAdminConsolePlugin(manifest.id);
```

Comment:
I think this has to be:
```suggestion
    new PluginRegistry(manifest.id).unregisterAdminConsolePlugin();
```

since `unregisterAdminConsolePlugin` doesn't take a parameter.

Reply:
Good catch. Done.